libsplinter/src/oauth/rest_api/actix/callback.rs

```diff
@@ -98,6 +106,27 @@ pub fn make_callback_route(
                                             HttpResponse::InternalServerError()
                                                 .json(ErrorResponse::internal_error())
                                         }
```

Comment:
The branches here are all being discarded because they aren't assigned to a variable or returned. The `Ok(_)` branch can just be `=> {}` and the `Err` branches should `return` their `HttpResponse`s.

Reply:
Fixed